krm-functions/cloudbuild.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: "bash"
+    args:
+      - "echo"
+      - "Cloud build substitution check: "
+      - "PROJECT_ID=$PROJECT_ID"
+      - "TAG_NAME=$TAG_NAME"
+  - name: ubuntu
+    entrypoint: bash
+    args:
+      - -c
+      - |
+        echo $TAG_NAME | sed 's:.*v::' | tr -d '"' > /workspace/image_tag.txt
```

Comment:
would `basename` work for this?

Reply:
Ah, probably, I can try that. That's probably a cleaner and more readable solution.